### PR TITLE
Preserve data dumps by default following replication script

### PR DESCRIPTION
**Pull request by edwardkerry**
Currently data dumps are automatically deleted following each sync script.
Should there be a failure at any point, they must all then be re-downloaded.
This can be extremely time consuming if experiencing issues with the VM.

This commit replaces the `-k` flag with `-x` to make the data dumps persist by default, and
only be automatically deleted if the flag is specified.

This will mean large amounts of data will be kept on developer's machines until manually deleted,
but will prevent having to re-download many GB's of data dumps should the VM import process fail.

### CODE REVIEW

File: development-vm/replication/common-args.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ OPTIONS:
     -q       Skip MySQL import
     -e       Skip Elasticsearch import
     -t       Skip Mapit import
-    -k       Keep backups after import (will delete by default)
+    -k       Delete backups after import (will keep by default)
```
Comment (deborahchua) [inline development-vm/replication/common-args.sh]: I'm wondering whether we should use a different letter, as most of us may not initially realise this has changed and still include `-k` when we want to keep the backups and then realised after it's deleted them!

Comment (deborahchua) [reply development-vm/replication/common-args.sh]: maybe `del` as `d` is already taken?

Comment (edwardkerry) [reply development-vm/replication/common-args.sh]: nice, `d` was gone but `del` is a good idea!

Comment (edwardkerry) [reply development-vm/replication/common-args.sh]: sadly, that is parse as `-d el` so sets the directory as `el`

Comment (deborahchua) [reply development-vm/replication/common-args.sh]: Ah! How about `x`?

**Pull request closed without merging**